scripts/replace-import-for-cjs.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e # exit on error
+
+# replace all imports of instantsearch.js with /cjs
+find dist/cjs -type f -exec sed -i.bak 's|instantsearch.js/es|instantsearch.js/cjs|g' {} +
```

Comment:
What's this for? I thought we changed imports from `from 'instantsearch.js'` to `from 'instantsearch.js/es'`.

( although `from 'instantsearch.js'` is still used for type imports)

Reply:
cjs can not import esm, so inside the cjs build all /es needs to be changed to /cjs

Reply:
I understand now, I had misread the script.